src/app/pages/waiting-room-to-car/cat-adi-part2/components/accompaniment-card/accompaniment-card.cat-adi-part2.ts

```diff
@@ -0,0 +1,46 @@
+import {
```

Comment:
I think this component can be removed as there is a component doing same thing inside ```waiting-room-to-car/components``` called ```accompaniment-card```